helm/values.yaml

```diff
@@ -0,0 +1,256 @@
+# Default values for authelia.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: clems4ever/authelia
+  tag: v3.16.2
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+authelia:
+  logLevel: "debug"
+  redirectionURL: "https://home.example.com:8080/"
+  totpIssuer: "authelia.com"
+
+  duo:
+    enabled: false
+    config:
+      hostname: api-123456789.example.com
+      integration_key: ABCDEF
+      secret_key: 1234567890abcdefghifjkl
+
+  ldap:
+    enabled: false
+    config:
+      url: ldap://127.0.0.1
+      # The base dn for every entries
+      base_dn: dc=example,dc=com
+      # An additional dn to define the scope to all users
+      additional_users_dn: ou=users
+      # The users filter used to find the user DN
+      # {0} is a matcher replaced by username.
+      # 'cn={0}' by default.
+      users_filter: cn={0}
+      # An additional dn to define the scope of groups
+      additional_groups_dn: ou=groups
+      # The groups filter used for retrieving groups of a given user.
+      # {0} is a matcher replaced by username.
+      # {dn} is a matcher replaced by user DN.
+      # {uid} is a matcher replaced by user uid.
+      # 'member={dn}' by default.
+      groups_filter: (&(member={dn})(objectclass=groupOfNames))
+      # The attribute holding the name of the group
+      group_name_attribute: cn
+      # The attribute holding the mail address of the user
+      mail_attribute: mail
+      # The username and password of the admin user.
+      user: cn=admin,dc=example,dc=com
+      password: password
+
+    # If LDAP is disabled, you can specify a users database file, however
```

Comment:
I know understand your concerns about the read-only characteristic of the file. Let's takle that later in v4.